src/utilities.cpp

```diff
@@ -37,5 +37,34 @@ void write_file_to_flash(const char *filename, uint32_t* counter) {
 
     file.print(*counter);
 
+    file.close();
+}
+
+uint8_t read_gdo_security_from_flash(const char* filename) {
```

Comment:
This function is nearly identical to read_file_from_flash. With a different default file value.  Can we use a common function?

Reply:
done tweaked read to allow a default value